app/src/main/java/com/sun/dummyshop/DummyShopApplication.kt

```diff
@@ -0,0 +1,25 @@
+package com.sun.dummyshop
+
+import android.app.Application
+import com.sun.dummyshop.di.billRepositoryModule
+import com.sun.dummyshop.di.databaseModule
+import com.sun.dummyshop.di.productRepositoryModule
+import org.koin.android.ext.koin.androidContext
+import org.koin.core.context.startKoin
+
+class DummyShopApplication : Application() {
+
+    override fun onCreate() {
+        super.onCreate()
+        startKoin {
+            androidContext(this@DummyShopApplication)
+            modules(
+                listOf(
```

Comment:
hình như truyền vararg được, không cần bỏ vào list 
```suggestion
            modules(
```

Reply:
vâng